src/user_interface.py

```diff
@@ -39,3 +39,28 @@ def handle_incorrect_input(self, user_input):
             valid_move = self.input_in_range(user_input)
             break
         return user_input
+
+    def get_play_again_user_input(self):
+        play_again_input = input()
+        is_valid_input = self.valid_play_again_input(play_again_input)
+        if is_valid_input:
+            if play_again_input.upper() == "Y":
+                return "Y"
+            else:
+                return "N"
```

Comment:
why return "Y" or "N" instead of `True` of `False`?

Reply:
I actually did go back and forth returning `True`/`False` and then this way.  I couldn't get it to work with True/False but I will try again with a different approach!

Reply:
Fixed!

Reply:
Actually I reverted back to the Y/N.  I tried it with True/False but it will error because the validation is checking for Y/N, and if `get_play_again_user_input` returns T/F, the validation will error.  I've been at it every which way to see if I could make it work, so I'm open to it but I haven't found a way that won't error!